utils/Makefile

```diff
@@ -0,0 +1,52 @@
+POETRY := poetry run
+MISS_HIT_STYLE := $(POETRY) mh_style
+MISS_HIT_LINT := $(POETRY) mh_lint
+MISS_HIT_METRIC := $(POETRY) mh_metric
+
+# Directories to analyze
+DIRECTORIES := ./../docs/ ./../test/
+# TODO: Add docs examples and src
+
+.PHONY: all
+all: check
+
+.PHONY: install
+install:
+	@echo "Installing dependencies..."
+	poetry install
```

Comment:
In general, I prefer poetry than pip as it is modern easy to use as poetry ensures consistent and reproducible builds with its `poetry.lock` file and simplifies dependency and project management in a single `pyproject.toml`. Ref: https://nanthony007.medium.com/stop-using-pip-use-poetry-instead-db7164f4fc72. 

Should be enough to install it https://python-poetry.org/docs/ and run `make install` but maybe something else is needed to make sure it creates a local `.venv` folder as regular pip with all deps.

Reply:
ok by me!